pkg/minikube/machine/client.go

```diff
@@ -183,7 +186,21 @@ func (api *LocalClient) Create(h *host.Host) error {
 	}{
 		{
 			"bootstrapping certificates",
-			func() error { return cert.BootstrapCertificates(h.AuthOptions()) },
+			func() error {
+				// CA cert and client cert should be generated atomically, otherwise might cause bad certificate error
+				lockErr := api.flock.LockWithTimeout(time.Second * 5)
```

Comment:
add issue link here why we do this...say this causes docker-env in parallel work